lib/stdlib/src/sys.erl

```diff
@@ -658,6 +677,8 @@ debug_options([{log_to_file, FileName} | T], Debug) ->
     end;
 debug_options([{install, {Func, FuncState}} | T], Debug) ->
     debug_options(T, install_debug(Func, FuncState, Debug));
+debug_options([{install, {FuncId, Func, FuncState}} | T], Debug) ->
+    debug_options(T, install_debug(FuncId, {Func, FuncState}, Debug));
```

Comment:
This is just a thought... would it be better to use {FuncId,Func} as key instead of only FuncId? That way you would get some sort if namespacing for your ids. The drawback is that you would need to specify both the id and the fun when removing it again. Did you consider this when implementing? Any thoughts?

Reply:
> The drawback is that you would need to specify both the id and the fun when removing it again.
I prefer sending small things in message passing. If this was in a single process, using `{FuncId, Func}` was good. but we should get these from a client process and we should force client processes to keep `{FuncId, Func}` for removing too.

Reply:
Yes, I think that's ok then.